### PR TITLE
Frontend/#043 button and link components

### LINKED ISSUE #43: FRONTEND | Button Component

### PR DESCRIPTION
**Pull request by vieraboschkova**
closes #43

### CODE REVIEW

File: client/src/components/Button/index.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
 
-function Button({ textToDisplay }) {
-  return <button className="button">{textToDisplay}</button>;
+function Button({ textToDisplay, classes, callback }) {
+  return (
+    <button className={classes} onClick={callback}>
+      {textToDisplay}
+    </button>
+  );
 }
```
Comment (aleksanderwalczuk) [inline client/src/components/Button/index.js:9]: don't you think it's way more complicated to use buttons like this?

**Review by aleksanderwalczuk — APPROVED** [review]
In my opinion it's the seed to spaghetti code. I find this code way more difficult in usage and it doesn't bring anything more than standard `<button>` element.

File: client/src/components/Button/index.js (hunk 1 above)
Comment (synowa) [reply client/src/components/Button/index.js:9]: Could you suggest any specific solution to this problem?

Comment (aleksanderwalczuk) [reply client/src/components/Button/index.js:9]: @synowa just use buttons as standard jsx element. There's no need to extract it to a separate component right now.
Let's say:
```jsx
<codeBefore/>
<button anyPropsINeedRightNow>anyText</button>
<codeAfter/>
```
Pros:
1. No imports (less messy components directory in the future)
2. Every additional prop that's not defined in `Button` component would have to be declared

If your not able to define how many props you'll need then you should use the spread operator
```jsx
const Example = (props) => (
    <someElement ...props>{props.children}</someElement>
);
```

_props.children stands for any child you put inside this component, eg. text or another component_

Comment (vieraboschkova) [reply client/src/components/Button/index.js:9]: I actually had in mind a very specific button, with specific style and function - a go back button, using props.history.goBack(). But I didn't have time to implement it yet. So far yes, it is reduntant.

**Pull request closed without merging**